tests/kubernetes/test_pod_generator.py

```diff
@@ -631,6 +631,15 @@ def test_reconcile_specs(self):
         client_spec.active_deadline_seconds = 100
         assert client_spec == res
 
+    def test_reconcile_specs_init_containers(self):
+        base_objs = [k8s.V1Container(name='base_container1')]
+        client_objs = [k8s.V1Container(name='client_container1')]
+        base_spec = k8s.V1PodSpec(priority=1, containers=[], init_containers=base_objs)
+        client_spec = k8s.V1PodSpec(priority=2, containers=[], init_containers=client_objs)
```

Comment:
```suggestion
        base_spec = k8s.V1PodSpec(containers=[], init_containers=[k8s.V1Container(name='base_container1')])
        client_spec = k8s.V1PodSpec(containers=[], init_containers=[k8s.V1Container(name='client_container1')])
```

Do we need `priority`? Does this work instead?